src/pages/AddYoutubeVideos.jsx

```diff
@@ -15,6 +16,13 @@ function AddYoutubeVideos() {
 		}
 	}, [currentUser, navigate]);
 
+	useEffect(() => {
+		ReactGA.send({
+			hitType: 'page_view',
+			page_location: window.location.pathname,
+		});
+	});
```

Comment:
this would cause page_view event to be sent on every rerender, so you will have many more `page_view` events than ground truth.

Reply:
Add an empty dependency array to ensure the `page_view` is sent on component mount.

Reply:
(If that is what you intend)

Reply:
Please make the changes everywhere you have called ReactGA.send

Reply:
more over, why don't you call the `useEffect` in the layout component, and add window.location.pathname as the dependency array argument. This way you won't have to call it in every single component.